pkg/monitor/datapath_sock_trace.go

```diff
@@ -31,3 +40,49 @@ type TraceSockNotify struct {
 	L4Proto    uint8
 	Flags      uint8
 }
+
+func (t *TraceSockNotify) IsIpv6() bool {
+	return (t.Flags & TraceSockNotifyFlagIPv6) != 0
+}
+
+func (t *TraceSockNotify) DumpDebug(prefix string) {
+	buf := bufio.NewWriter(os.Stdout)
+
+	fmt.Fprintf(buf, "%s [%s] cgroup_id: %d sock_cookie: %d, dst [%s:%d] %s \n",
```

Comment:
using `bufio.Writer` doesn't seem to buy much here. Why not just use `fmt.Printf`?

Reply:
That seems reasonable to me. I followed the same style as the per packet events parser, but we don't necessarily need to follow the same convention as the data is fixed size, and relatively much smaller. @joestringer Is there any other reason why we should opt for a buffered writer in case of cilium monitor?

Reply:
Yeah I noticed this too while reviewing, and I figured that it's fine if this is consistent with the rest of the code. We could definitely do better on this, but I'd rather tackle that across all of these functions rather than incrementally changing it in this PR.